SauceExamples/Visual/UnitTest1.cs

```diff
@@ -0,0 +1,99 @@
+using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Remote;
+using System;
+using System.Collections.Generic;
+
+namespace WebDriver_CSharp_Example
+{
+    [TestFixture]
+    public class VisualE2ETest
+    {
+        RemoteWebDriver driver;
+        public string sauceUserName = Environment.GetEnvironmentVariable("SAUCE_USERNAME");
+        public string sauceAccessKey = Environment.GetEnvironmentVariable("SAUCE_ACCESS_KEY");
+        public string screenerApiKey = Environment.GetEnvironmentVariable("SCREENER_API_KEY");
+        public string seleniumProtocol = Environment.GetEnvironmentVariable("SELENIUM_PROTOCOL");
```

Comment:
As mentioned in another comment, let's simplify and remove the `SELENIUM_PROTOCOL`, `SELENIUM_HOST`, and `SELENIUM_PORT` environment variables and checks in this file